AlgorithmiaTests/AlgorithmiaTests.swift

```diff
@@ -132,4 +132,59 @@ class AlgorithmiaTests: XCTestCase {
         }
     }
     
+    func testFilePut() {
+        let expect = expectation(description: "Test put File")
+        let file = client?.file("data://.my/test/test.txt")
+        file?.put(string: "test text", completion: { (error) in
+            if let error=error {
+                XCTFail("Algorithmia File put Test error: \(error)")
+            } else {
+                expect.fulfill()
+            }
+        })
+        
+        waitForExpectations(timeout: 10.0) { error in
+            if let error = error {
+                XCTFail("WaitForExectationsWithTimeout error: \(error)")
+            }
+        }
+    }
+    
+    func testFileExits() {
+        let expect = expectation(description: "Test File exist")
+        let file = client?.file("data://.my/test/test.txt")
+        file?.exists(completion: { (isExist, error) in
+            if isExist {
```

Comment:
If an error occurred, I presume `isExist` be `nil`. It won't break this test, but it seems like checking for error in the case of `exists` is probably the better pattern?